app/bundles/EmailBundle/Model/EmailModel.php

```diff
@@ -2200,7 +2200,7 @@ public function sendSampleEmailToUser(
         );
         $this->dispatcher->dispatch($event, EmailEvents::EMAIL_ON_DISPLAY);
 
-        $mailer = $this->mailHelper->getSampleMailer();
+        $mailer = $this->mailHelper->getMailer();
```

Comment:
Shouldn't this be with the `true` for "clean slate"? That was the default value in the `getSampleMailer` method.
```suggestion
        $mailer = $this->mailHelper->getMailer(true);
```

Reply:
It is the default for the called method too.